tests/contrib/test_pydantic.py

```diff
@@ -55,14 +63,24 @@ class PydanticMetaOverride:
         self.maxDiff = None
 
     async def test_backward_relations(self):
+        # from meta_override:
```

Comment:
It would be cleaner to have two separate tests, for instance, test_backward_relations_with_meta_override and  test_backward_relations_with_pydantic_meta